src/history.js

```diff
@@ -2,25 +2,39 @@ import Action from './action'
 import Emitter from './emitter'
 
 import {
-  inherit
+  inherit,
+  merge
 } from './utils'
 
 import {
   BIRTH,
   START
 } from './lifecycle'
 
+const DEFAULTS = {
+  maxHistory: 1,
+
+  updater () {
+    return update => update()
+  }
+}
+
 /**
  * The central tree data structure that is used to calculate state for
  * a Microcosm. Each node in the tree is an action. Branches are
  * changes over time.
  * @constructor
  */
-export default function History (limit) {
+export default function History (config) {
```

Comment:
oh I see, sounds good. somehow I thought I was looking at the Microcosm constructor